q2_alignment/plugin_setup.py

```diff
@@ -57,8 +57,9 @@
                      'all available cores)',
         'parttree': 'This flag is required if the number of sequences being '
                     'aligned are larger than 1000000. Disabled by default',
-        'fragments': 'This flag indicates that alignment optimized for '
-                     'addition of fragmentary sequences should be used.'},
+        'addfragments': 'Optimize for the addition of short sequence '
+                        'fragments (for example, primer or amplicon '
+                        'sequences).'},
```

Comment:
I was just wondering if we should be a tiny bit more explicit to avoid confusion:

```
'addfragments': 'Optimize for the addition of short sequence '
                'fragments (for example, primer or amplicon '
                'sequences). If not set, default sequence addition '
                'is used.'
```

Reply:
True, good point - updated.